index.bs

```diff
@@ -0,0 +1,160 @@
+<pre class='metadata'>
+Title: Runtime Keys
+Shortname: runtime-keys
+Level: 1
+Status: w3c/CG-DRAFT
+Group: wintercg
+URL: http://runtime-keys.proposal.wintercg.org
+Editor: Ethan Arrowood, Vercel http://vercel.com, ethan-arrowood@vercel.com
+Abstract: A proposal defining standard identifier keys for various runtime environments
+Markup Shorthands: markdown yes
+</pre>
+<pre class=link-defaults>
+spec:url; type:interface; text:URL
+spec:html; type:attribute; for:Window; text:navigator
+spec:html; type:attribute; for:Window; text:self
+</pre>
+
+Introduction {#intro}
+=====================
+
+This proposal defines a list of keys that represent various runtime environments. These keys can be used in a variety of ways, and should be considered a reliable representation of the given runtime. For example, they may be used in a project configuration file to indicate the given project supports the specified runtime. This specification does not detail <i>how</i> the keys can be used; that is left up to community developed tools. The point of this proposal is too only define <i>what</i> the keys are in order to prevent conflicts and give users a reliable list of platforms they can build tooling around.
+
+Keys {#keys}
+============
```

Comment:
Just a couple of items here from the discussion today:

1. We need specific language added here that no endorsement of this spec or wintercg is implied by listing a company or organization or project in this registry. 
2. We need specific language added that indicates inclusion in this registry does not imply that the runtime is conformant with the wintercg minimum common api.

Reply:
Included two lines covering this detail. let me know what you think